physicalplan/scan/matrix_selector.go

```diff
@@ -170,9 +195,6 @@ func (o *matrixSelector) loadSeries(ctx context.Context) error {
 		o.series = make([]labels.Labels, len(series))
 		for i, s := range series {
 			lbls := s.Labels()
-			if o.funcExpr.Func.Name != "last_over_time" {
```

Comment:
Is this safe to do?

Reply:
This seems to be moved to https://github.com/thanos-community/promql-engine/pull/62/files#diff-c5565c814b1f8018448d49f7dc602d00b7cf0225383932129b1d5ba967bfafa0R257